scripts/make.py

```diff
@@ -526,3 +526,27 @@ def compiler_str(bin_name: str) -> str:
             raise util.ToolchainBuildError from ex
         if cfg.is_cross_compiling:
             self._copy_runtime_to_native(lib_spec)
+
+    def _run_smoke_tests(self, _lib_spec: config.LibrarySpec) -> None:
```

Comment:
The test only supports armv6m, so I think it does not make sense to run the tests for each library variant we build.